src/main/java/seedu/address/model/client/Client.java

```diff
@@ -67,6 +75,12 @@ public Address getAddress() {
         return address;
     }
 
+    public Optional<Birthday> getBirthday() {
+        return (birthday == null)
+                ? Optional.empty()
+                : Optional.of(birthday);
+    }
+
```

Comment:
As mentioned above, I think Client should not be the one to catch the `null` value here. should let the parser do the job instead.